Common/src/main/java/vazkii/botania/common/item/ModItems.java

```diff
@@ -304,6 +304,7 @@
 	public static final Item sparkUpgradeIsolated = new ItemSparkUpgrade(/*ModPatterns.SPARK_ISOLATED, */defaultBuilder(), SparkUpgradeType.ISOLATED);
 	public static final Item corporeaSpark = new ItemCorporeaSpark(defaultBuilder());
 	public static final Item corporeaSparkMaster = new ItemCorporeaSpark(defaultBuilder());
+	public static final Item corporeaSparkCreative = new ItemCorporeaSpark(defaultBuilder());
```

Comment:
```suggestion
	public static final Item corporeaSparkCreative = new ItemCorporeaSpark(defaultBuilder().rarity(Rarity.EPIC));
```
For consistency with all the creative mode items.